src/main/java/org/gridsuite/voltageinit/server/computation/service/AbstractResultContext.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.voltageinit.server.computation.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.Getter;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.support.MessageBuilder;
+
+import java.io.UncheckedIOException;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+
+import static org.gridsuite.voltageinit.server.computation.service.NotificationService.*;
+
+/**
+ * @author Mathieu Deharbe <mathieu.deharbe at rte-france.com>
+ * @param <R> run context specific to a computation, including parameters
+ */
+@Getter
+public abstract class AbstractResultContext<R extends AbstractComputationRunContext<?>> {
+
+    protected static final String RESULT_UUID_HEADER = "resultUuid";
+
+    protected static final String NETWORK_UUID_HEADER = "networkUuid";
+
+    protected static final String REPORT_UUID_HEADER = "reportUuid";
+
+    public static final String VARIANT_ID_HEADER = "variantId";
+
+    public static final String REPORTER_ID_HEADER = "reporterId";
+
+    public static final String REPORT_TYPE_HEADER = "reportType";
+
+    protected static final String MESSAGE_ROOT_NAME = "parameters";
+
+    protected final UUID resultUuid;
+
+    protected final R runContext;
```

Comment:
no need this extra visibility
```suggestion
    private final UUID resultUuid;
    private final R runContext;
```

Reply:
I thought the extra visibility was needed in one of the other computation microservices but I didn't find it. So probably true.